ui/src/lib/services/sseService.ts

```
@@ -388,6 +392,24 @@ export class ChatService {
         this.callbacks.onToken?.(event.text, event.conversation_id);
         break;
 
+      case "token.usage":
+        const usageEvent = event as TokenUsageEvent;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)